File: Java/Desktop/NewLab8/src/by/bntu/fitr/povt/cng/model/entity/Garland.java
package by.bntu.fitr.povt.cng.model.entity;

import by.bntu.fitr.povt.cng.model.entity.abstracts.Toy;

import java.io.Serializable;

public class Garland extends Toy implements Serializable {
    private double length;
    private int countOfElements;
    private String element;

    public Garland() {
    }

    public Garland(double price, double weight, double size, double length, int countOfElements, String element) {
        super(price, weight, size);
        this.length = length;
        this.countOfElements = countOfElements;
        this.element = element;
    }

    public double getLength() {
        return length;
    }

    public void setLength(double length) {
        this.length = length;
    }

    public int getCountOfElements() {
        return countOfElements;
    }

    public void setCountOfElements(int countOfElements) {
        this.countOfElements = countOfElements;
    }

    public String getElement() {
        return element;
    }

    public void setElement(String element) {
        this.element = element;
    }

    @Override
    public String toString() {
        return new StringBuilder().append("\nGarland ").append(super.toString()).append("\n\tLength: ").append(length).append("\n\tCount of elements: ").append(countOfElements).append("\n\tElement: ").append(element).toString();//length,countofelem,elem
    }

}

File: Csharp/Lab8/Lab8/Lab8/cng/Model/Entity/Garland.cs
using System;
using System.Text;
using Lab8.cng.Model.Entity.Abstracts;

namespace Lab8.cng.Model.Entity
{
    public class Garland : Toy
    {
        private double _length;
        private int _countOfElemets;
        private string _element;

        public Garland() {
        }

        public Garland(double price, double weight, double size, double length, int countOfElemets, string element) : base(price, weight, size) {
            this._length = length;
            this._countOfElemets = countOfElemets;
            this._element = element;
        }

        public double GetLength() {
            return _length;
        }

        public void SetLength(double length) {
            this._length = length;
        }

        public int GetCountOfElemets() {
            return _countOfElemets;
        }

        public void SetCountOfElemets(int countOfElemets) {
            this._countOfElemets = countOfElemets;
        }

        public String GetElement() {
            return _element;
        }

        public void SetElement(String element) {
            this._element = element;
        }

        
        public override string ToString() {
            return new StringBuilder().Append("\nGarland " + base.ToString() + "\n\tLength: " + _length + "\n\tCount of elements: " + _countOfElemets + "\n\tElement: " + _element).ToString();//length,countofelem,elem
        }
    }
}